app/views/users/show.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= link_to "Back to Event page", events_path %>
+
+<% if @user.attended_events.empty? %>
+<%= "The user doesnt attended any event" %>
+<% end %>
+<p><strong>Created Events by You</strong></p>
+
+<% @created_events.each do |event| %>
+<ul>
+<li>
+<%= event.description %>
+</li>
+</ul>
+<% end %>
+<br>
+<p><strong> Past Event</strong></p>
+<% if @past_events.empty? %>
+<%= "There is no past event attended" %>
+<% else %>
+<% @past_events.each do |past_events| %>
+<ul>
+<li>
+<%= past_events.description %>
+</li>
+</ul>
+<% end %>
+<% end %>
+<p><strong> Up coming Event</strong></p>
+<% if @upcoming_events.empty? %>
+<%= "The there is no upcoming event attended" %>
+<% else %>
+<% @upcoming_events.each do |upcoming_events| %>
+<ul>
+<li>
+<%= upcoming_events.description %>
+</li>
+</ul>
```

Comment:
**[OPTIONAL]**
Note! this is considered optional because the previous reviewer may have missed it.
- [ ] There should be absolutely NO logic in views. Views are only for displaying information that became available to it from its corresponding controller not generating it.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright)

**Don't know where to start or how to?** See  [Rails View Helpers Best Practices](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4)